include/vrv/vrv.h

```diff
@@ -55,7 +55,7 @@ LogLevel StrToLogLevel(const std::string &level);
 /**
  * Utility for comparing doubles
  */
-bool AreEqual(double dFirstVal, double dSecondVal);
+bool AreNearlyEqual(double dFirstVal, double dSecondVal);
```

Comment:
`ApproximatelyEqual` or `AboutEqual` might be better name for `AreNearlyEqual` (two words rather than three).

Reply:
Yes, sounds great.